src/test/ui/async-await/await-into-future.rs

```diff
@@ -0,0 +1,28 @@
+// check-pass
```

Comment:
I think I'd prefer to have one run-pass test for this feature.

In case you don't know, this directory has some utils in auxiliary/ to make this easier, see sibling tests for examples.